content/source/docs/cloud/users-teams-organizations/api-tokens.html.md

```diff
@@ -0,0 +1,87 @@
+
+## Access Levels
+
+The following chart illustrates the various access levels for the supported API token types. Some permissions are implicit based on the token type, others are dependent on the permissions of the associated user, team, or organization.
+
+🔷 = Implicit for token type 🔶 = Requires explicit permission
+
+|                                | User tokens | Team tokens | Organization tokens |
+|--------------------------------|:-----------:|:-----------:|:-------------------:|
+| **Users**                      |             |             |                     |
+| Manage user settings           | 🔷          |             |                     |
+| Manage user tokens             | 🔷          |             |                     |
+| **Workspaces**                 |             |             |                     |
+| Read workspace variables       | 🔶          | 🔶          | 🔷                  |
+| Write workspace variables      | 🔶          | 🔶          | 🔷                  |
+| Plan, apply, upload states     | 🔶          | 🔶          |                     |
+| Force cancel runs              | 🔶          | 🔶          |                     |
+| Create configuration versions  | 🔶          | 🔶          |                     |
+| Create or modify workspaces    | 🔶          | 🔶          | 🔷                  |
+| Remote operations              | 🔶          |             |                     |
+| **Teams**                      |             |             |                     |
+| Create teams                   | 🔶          |             | 🔷                  |
+| Modify team                    | 🔶          | 🔷          | 🔷                  |
+| Read team                      | 🔶          | 🔷          | 🔷                  |
+| Manage team tokens             | 🔶          | 🔷          | 🔷                  |
+| Manage team workspace access   | 🔶          | 🔶          | 🔷                  |
+| Manage team membership         | 🔶          | 🔷          | 🔷                  |
+| **Organizations**              |             |             |                     |
+| Create or modify organizations | 🔶          |             |                     |
+| Manage organization tokens     | 🔶          |             |                     |
+| **Sentinel**                   |             |             |                     |
+| Manage Sentinel policies       | 🔶          | 🔶          | 🔷                  |
+| Manage policy sets             | 🔶          | 🔶          | 🔷                  |
+| **Integrations**               |             |             |                     |
+| Manage VCS connections         | 🔶          | 🔶          | 🔷                  |
+| Manage SSH keys                | 🔶          | 🔶          | 🔷                  |
+| **Modules**                    |             |             |                     |
+| Manage Terraform modules       | 🔶          | 🔶          |                     |
```

Comment:
Whooooa this is doooope. 💯 

`Override policy checks` is one that should be added here, we get asked that one a lot.

If `Manage organization tokens` is here, might want to add `Manage team tokens` (pretty sure I added managing team tokens with org tokens a while back 😑) (though IMO should just leave both out, maybe)

Reply:
👍 on `Override policy checks`, good call.

`Manage team tokens` is already in here, under `Teams`!